homeassistant/components/light/wemo.py

```diff
@@ -144,10 +163,42 @@ def turn_off(self, **kwargs):
         """Turn the light off."""
         transitiontime = int(kwargs.get(ATTR_TRANSITION, 0))
         self.wemo.turn_off(transition=transitiontime)
+        self._is_on = False
```

Comment:
Remove this too.

Reply:
Good catch. Those where some carry-overs from the original code.